index.html

```diff
@@ -0,0 +1,176 @@
+       <li class="button">html</li>
+       <li class="skil button">bootstrap</li>
+       <li class="skil button">Ruby</li>
+    </ul>
+    </div>
+    <a class="see-project-2 button" href="">See Project</a>
+    
+</section>
+
+<section class="section-3">
+   <div class="about-me">
+      <h1 class="h31">About me</h1>
+      <p class="p31">Hello I’m a software developer! I can help you build a product ,feature or 
+         website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <a class="get-myresume button" href="">Get My Resume</a>
+      <img class="abt-me-image" src="./assets/images/Group1.png" alt="">
+   </div>
+   <hr class="line-3">
+
+   <ul class="lists">
+      <li class="Lan">Language 
+          <ul class="lan-list">
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>Html</li>
+              <li>Css</li>
+            </ul>
+      </li>
+        
+      <li class="Frame"> Framework
+          <ul class="Frame-list">
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+              <li>CapyBara</li>
+              <li>Selenium</li>
+          </ul>
+      </li>
+      <li class="Sk">Skills
+          <ul class="Sk-list">
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+      </li>
+  </ul>
+</section>
+        <form class="upper-form" action="https://formspree.io/f/xoqrrprd" method="post">
+         <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch. </p>
+            <input type="text" maxlength="30" required placeholder="Full Name" class="full-name" name="usrname">
+            <input type="email" required placeholder="Email" class="email" name="usremail">
+            <textarea maxlength="500" id="msg" placeholder="Enter text here" class="messages" name="usrmessages"></textarea>
+            <input type="submit" value="Get it touch" class="touch" name="usrsubmit">
+        </form>
+
+<footer class="footer-social-media">
+  <ul class="social-links">
+      <li class="socia-git"><a href="https://github.com/solog0039"><img  src="./assets/social media footer/Icon -GitHub.png" alt="Github"></a></li>
+      <li class="social-Linkdinf"><a href="www.linkedin.com/in/solomon-kidanu-62a994232"><img  src="./assets/social media footer/Icon - Linkedin.png" alt="linkedin"></a></li>
+      <li class="social-c"><a href="##"><img  src="./assets/social media footer/Angelist.png" alt="tum"></a></li>
+      <li class="social-c"><a href="https://twitter.com/Solomon57320119"><img  src="./assets/social media footer/Icon - Twitter.png" alt="tiwtter"></a></li>
+      <li class="social-c"><a href="##"><img  src="./assets/social media footer/medium.png" alt="Medium"></a></li>
+  </ul>
+</footer>
```

Comment:
Good job with the top border. But let's make sure we don't create new issues while we are fixing the old ones. The line needs to be implemented across the screen. Kindly fix that. 🦾
Figma template: 
<img width="392" alt="Screenshot 2023-01-25 at 11 15 54 PM" src="https://user-images.githubusercontent.com/25887253/214681420-b40c9c8c-9805-4e7f-a843-42d93781b084.png">
Your design:
<img width="392" alt="Screenshot 2023-01-25 at 11 16 18 PM" src="https://user-images.githubusercontent.com/25887253/214681426-c6c90852-e811-45e4-8410-0843cc6f0ac0.png">